chsdi/models/vector/bafu.py

```diff
@@ -2357,7 +2357,18 @@ class LandesForstInventarVegetationsHoehenModell(Base, LandesForstInventarVegeta
 register(LandesForstInventarVegetationsHoehenModell.__bodId__, LandesForstInventarVegetationsHoehenModell)
 
 
-class LandesForstInventarVegetationsHoehenModellRelief(Base, LandesForstInventarVegetation, Vector):
+class LandesForstInventarVegetationRelief:
+    __tablename__ = 'vegetationshoehenmodell_relief'
+    __table_args__ = ({'schema': 'wald', 'autoload': False, 'extend_existing': True})
+    __template__ = 'templates/htmlpopup/landesforstinventar-vegetationshoehenmodell.mako'
+    __queryable_attributes__ = ['datenstand']
```

Comment:
please don't use this anymore, if the attribute is defined as searchable in DIF, we are creating a Sphinx index.

Reply:
see here: https://github.com/geoadmin/mf-chsdi3/pull/3691

Reply:
@faselm can you remove the querryable attributes, thanks!